supabase/migrations/20240528091033_add_completed_parcels_view.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE VIEW completed_parcels AS
+WITH completed_events AS (
+    SELECT
+        events.parcel_id,
```

Comment:
The following parcels are counted ->
Parcels that have (at some point) had the event "Delivered", "Parcel Collected", "Fulfilled with Trussell Trust" AND the most recent event is NOT "Parcel Deleted" - they are dated in the reports table by the timestamp of the most recent time that they have been set to "Delivered", "Parcel Collected" or "Fulfilled with Trussell Trust".

Reply:
Dating by the time that they are delivered/collected makes the most sense to me as we only care about delivered/collected parcels, but I might be wrong on that assumption.